makefile

```diff
@@ -8,8 +8,14 @@ SOURCE_URL=https://icd11files.blob.core.windows.net/tmp/whofic-2023-04-08.owl.gz
 # MAIN COMMANDS / GOALS ------------------------------------------------------------------------------------------------
 all: tmp/output/release/icd11foundation.owl
 
+clean:
+	rm -rf tmp/
+
+tmp/output/release/icd11foundation.owl: tmp/output/intermediate1-unicode-cleaned.owl
+	robot remove --axioms equivalent -i $< -o $@
```

Comment:
## The essential change
This is the command that applies the fix.

- [ ] @joeflack4 Run `build-mondo-ingest` using the output from this to verify if it worked.

Reply:
I agree, I think that line (`robot remove --axioms equivalent -i $< -o $@`) is fine. 

I'm not sure if https://robot.obolibrary.org/remove#preserving-the-structure is also needed. I think that was needed with something in the Mondo repo in Sep 2023, but I don't recall the details or am able to find it in the repo files yet. I am leaving this note here as a reminder in case things do not look as expected.

Reply:
I will look at your comment tomorrow, but I just wanted to report back here that [the build passed](https://github.com/monarch-initiative/mondo-ingest/pull/412#discussion_r1496830887)!